Sources/BraintreePayPalNativeCheckout/BTPayPalNativeCheckoutClient.swift

```diff
@@ -31,7 +31,9 @@ import PayPalCheckout
         with nativeRequest: BTPayPalRequest,
         completion: @escaping (BTPayPalNativeCheckoutAccountNonce?, Error?) -> Void
     ) {
+        apiClient.sendAnalyticsEvent("ios.paypal-native-checkout.tokenize.started")
```

Comment:
Can we align with the existing BTNXO analytic event naming on Android? [Example here](https://github.com/braintree/braintree_android/blob/6e003485313db9eafe32fb55966ddeed444da46f/PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutClient.java#L113). 

So `paypal-native` instead of `paypal-native-checkout`? This will help us when searching for analytics across platforms.

Reply:
Good call - will update this